pandas/io/formats/style.py

```diff
@@ -72,32 +73,46 @@ class Styler(object):
 
     Notes
     -----
-    Most styling will be done by passing style functions into
-    ``Styler.apply`` or ``Styler.applymap``. Style functions should
-    return values with strings containing CSS ``'attr: value'`` that will
-    be applied to the indicated cells.
 
-    If using in the Jupyter notebook, Styler has defined a ``_repr_html_``
-    to automatically render itself. Otherwise call Styler.render to get
-    the genterated HTML.
-
-    CSS classes are attached to the generated HTML
-
-    * Index and Column names include ``index_name`` and ``level<k>``
-      where `k` is its level in a MultiIndex
-    * Index label cells include
-
-      * ``row_heading``
-      * ``row<n>`` where `n` is the numeric position of the row
-      * ``level<k>`` where `k` is the level in a MultiIndex
-
-    * Column label cells include
-      * ``col_heading``
-      * ``col<n>`` where `n` is the numeric position of the column
-      * ``evel<k>`` where `k` is the level in a MultiIndex
-
-    * Blank cells include ``blank``
-    * Data cells include ``data``
+    `Styler` renders HTML on a `DataFrame` using its own `Styler.render`
```

Comment:
I don't follow "renders HTML on a DataFrame". Perhaps "renders an HTML representation of a DataFrame"?